scripts/mods/mod_reforged/dynamic_spawns/parties/barbarian_parties.nut

```diff
@@ -16,14 +16,19 @@ local parties = [
 		function generateIdealSize( _isLocation = false )
 		{
 			local startingResources = this.getSpawnProcess().getStartingResources();
-			if (startingResources >= 216)
+
+			if (startingResources >= 550)
 			{
 				return 12;
 			}
-			else if (startingResources >= 164)
+			else if (startingResources >= 450)
 			{
 				return 10;
 			}
+			else if (startingResources >= 216)
+			{
+				return 8;
+			}
 			else
 			{
 				return 8;
```

Comment:
That's a lot of magic numbers, what are they based on? Shouldn't that be based on some entity cost?
Also last two conditions return the same value.